pkg/api/v1beta3/dynakube/extensions.go

```diff
@@ -102,3 +104,45 @@ type ImageRefSpec struct {
 	// Indicates a tag of the image to use
 	Tag string `json:"tag,omitempty"`
 }
+
+type LogAgentDaemonSetSpec struct {
```

Comment:
```suggestion
type LogAgentSpec struct {
```

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/3595/commits/a8bbce9077f2aa4e354807686d30ae72e8518cd4